components/Card/fragments/MicroService/Microservice.tsx

```diff
@@ -1,12 +1,31 @@
 import React from 'react';
-import { Grid } from '@mui/material';
+import { Button, CardActionArea, Grid } from '@mui/material';
+import { useRouter } from 'next/router';
 import { MicroserviceContainer } from './styles';
 import { MicroServiceProps } from './interfaces';
 
-export const Microservice = ({ serviceName }: MicroServiceProps) => (
-  <Grid item xs={12} sm={4} md={6} lg={4}>
-    <MicroserviceContainer>
-      {serviceName}
-    </MicroserviceContainer>
-  </Grid>
-);
+export function Microservice({ serviceName, isMicroservice }: MicroServiceProps) {
+  const router = useRouter();
+  function handelClick() {
+    router.push(`/project/${serviceName}`, undefined, { shallow: false }).then(() => console.log('redirected')).catch((e) => console.error(e));
+  }
+  if (isMicroservice) {
+    return (
+      <Grid item xs={12} sm={4} md={6} lg={4}>
+        <CardActionArea component={Button} onClick={() => handelClick()}>
+          <MicroserviceContainer>
+            {serviceName}
+          </MicroserviceContainer>
+        </CardActionArea>
+      </Grid>
+    );
+  }
+
+  return (
+    <Grid item xs={12} sm={4} md={6} lg={4}>
+      <MicroserviceContainer elevation={1}>
+        {serviceName}
+      </MicroserviceContainer>
+    </Grid>
+  );
+}

```

Comment:
Duplicated code, ich würde die beide kombinieren

```suggestion
  function handelClick() {
  if(isMicroservice){
    router.push(`/project/${serviceName}`)
    }
  }
    
    return (
      <Grid item xs={12} sm={4} md={6} lg={4}>
        <CardActionArea component={Button} onClick={() => handelClick()}>
          <MicroserviceContainer>
            {serviceName}
          </MicroserviceContainer>
        </CardActionArea>
      </Grid>
    );
  }
}

```